src/index.html

```diff
@@ -11,7 +11,152 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      >
+      <fieldset class="pesonal_information">
+        <legend>Personal information</legend>
+        <label class="pesonal_information-input">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          required
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
            required
          >
```
check all code style cases